lib/config/options/index.ts

```diff
@@ -1272,6 +1272,18 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'matchReleaseAge',
```

Comment:
We still need to:

- Explain how `matchReleaseAge` is different from `matchCurrentAge`.
- Explain how to decide if you need `matchReleaseAge` or `matchCurrentAge`.
- Give examples of cases where using th.e new feature is helpful.
- Give examples where the new feature is _worse_ than using `matchCurrentAge`.

Please add the explanations to the `docs/usage/configuration-options.md` file.

Reply:
I think I've covered what you mentioned, but I'm open to any suggestions on how it can be improved.